lib/components/template/layer/ImageLayerEditor.vue

```diff
@@ -93,7 +93,8 @@ v-expansion-panel#image-layer-editor(popout)
       dimensions() { return this.getLayerDimensions(this.layer) },
 
       templateSize() {
-        return this.findTemplateByLayerId(this.layer.id).size
+        let template = this.findTemplateByLayerId(this.layer.id)
```

Comment:
use `const` when you aren't overwriting the variable

Reply:
Done